mosec/mixin/typed_worker.py

```diff
@@ -27,32 +27,54 @@
     warnings.warn("msgpack is required for TypedMsgPackMixin", ImportWarning)
 
 
-def parse_forward_input_type(func):
+def parse_forward_input_type(func, target="parameters", index=0):
     """Parse the input type of the forward function.
 
     - single request: return the type
     - batch request: return the list item type
     """
     sig = inspect.signature(func)
-    params = list(sig.parameters.values())
-    if len(params) < 1:
-        raise TypeError("`forward` method doesn't have enough(1) parameters")
+    if target == "parameters":
+        params = list(sig.parameters.values())
+        if len(params) < 1:
+            raise TypeError("`forward` method doesn't have enough(1) parameters")
+        typ = params[index].annotation
+    else:
+        typ = sig.return_annotation
+        if typ is inspect.Signature.empty:
+            raise TypeError("`forward` method doesn't have return annotation")
```

Comment:
Why would an empty response annotation raise an exception?

Reply:
I want to inform users you can and shall annotate the type to enable this feature, at least a Any type. It's also ok if err not raised.